src/components/modal/Modal.tsx

```diff
@@ -0,0 +1,25 @@
+import "./modal.scss"
+import {useState} from "react";
+
+const Modal = () => {
+  const [visible, setVisible] = useState('');
```

Comment:
1. Change naming [visible, setVisible] -> [isVisible, setIsVisible]
2. it is better to have it boolean type:

const [isVisible, setIsVisible] = useState(false)